src/Disks/S3/DiskS3.h

```diff
@@ -199,14 +204,16 @@ class DiskS3 : public IDisk
     int list_object_keys_size;
 
     /// Key has format: ../../r{revision}-{operation}
-    const re2::RE2 key_regexp {".*/r(\\d+)-(\\w+).*"};
+    const re2::RE2 key_regexp {".*/r(\\w+)-(\\w+).*"};
```

Comment:
I didn't found where we use revision with alphabet chars.

Reply:
Seems it's my debbuging artifact :), will change back.